openmetadata-ui/src/main/resources/ui/src/components/common/DBTConfigFormBuilder/DBTCloudConfig.tsx

```diff
@@ -140,6 +144,23 @@ export const DBTCloudConfig: FunctionComponent<Props> = ({
           onChange={(e) => handleDbtCloudProjectId(e.target.value)}
         />
       </Field>
+      <Field>
+        <label className="block tw-mb-1 tw-form-label" htmlFor="dbtCloudJobId">
+          {t('label.dbt-cloud-job-id')}
+        </label>
+        <p className="text-grey-muted m-t-xss m-b-xs text-xs">
```

Comment:
Can we use `Typography` here instead of `<p>` tag?

Reply:
just to avoid inconsistency in the visual part I haven't changed anything here, We will altogether going to update this file, with ANTD component and going to remove the tailwind